src/Controller/Component/PrgComponent.php

```diff
@@ -21,6 +18,8 @@ class PrgComponent extends Component
      * - `queryStringBlacklist` : An array of form fields that should not end up in the query.
      * - `emptyValues` : A map of fields and their values to be considered empty
      *   (will not be passed along in the URL).
+     * - `modelClass` : Configure the controller's modelClass to be used for the query, used to
+     *   populate the _isSearch view variable to allow for a reset button, for example.
```

Comment:
Please add that setting this config to `false` will disable the auto setting of view var.

Reply:
Done.